gradle-sls-packaging/src/main/groovy/com/palantir/gradle/dist/service/gc/GcProfile.java

```diff
@@ -38,6 +38,10 @@
     class Throughput implements GcProfile {
         @Override
         public final List<String> gcJvmOpts(JavaVersion javaVersion) {
+            // For Java 15 and above, use G1 as the default garbage collector
+            if (javaVersion.compareTo(JavaVersion.toVersion("14")) > 0) {
```

Comment:
I think we want to change the default GCProfile that is used rather than changing the behaviour of `throughput` when on Java 15+

Reply:
Makes sense. Updated the PR to use `hybrid` for java >= 15 and `throughput` for java < 15.